frontend/src/modules/Survey/Components/StepBegin.tsx

```diff
@@ -40,53 +47,75 @@ export const StepBegin = ({
-              setEmail(e.target.value)
-            }
-            error={!isValidEmail}
-            helperText={isValidEmail ? ' ' : 'Invalid Email'}
-          />
-        </StyledFields>
-        <Box
-          sx={{
-            marginLeft: 'auto',
-            textAlign: 'center',
-            color: 'purple.100',
-            weight: 600,
-          }}
-        >
-          <IconButton
-            className="next"
-            onClick={gotoNextStep}
-            disabled={name === '' || email === '' || !isValidEmail}
-            sx={{ boxShadow: 3 }}
+        <main>
+          <StyledTitleWrapper>
+            <StyledHeaderText>Hi,</StyledHeaderText>
+            <StyledWelcomeText>Find study partners!</StyledWelcomeText>
+          </StyledTitleWrapper>
+
+          <StyledFields>
+            <InputLabel htmlFor="user name">
+              {' '}
+              <StyledLabelText> Name: </StyledLabelText>{' '}
+            </InputLabel>
+            <TextField
+              id="user name"
+              variant="standard"
+              aria-label="user name"
+              placeholder="Martha E. Pollack"
+              // variant="standard"
+              sx={textInputStyle}
+              value={name}
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                setName(e.target.value)
+              }
+            />
+            <InputLabel htmlFor="user email">
+              {' '}
+              <StyledLabelText> Email: </StyledLabelText>{' '}
+            </InputLabel>
+            <TextField
+              id="user email"
+              variant="standard"
+              aria-label="user email"
+              value={email}
+              sx={textInputStyle}
+              type="email"
+              onBlur={() => setIsValidEmail(validEmail.test(email))}
+              placeholder="mep22@cornell.edu"
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                setEmail(e.target.value)
+              }
+              error={!isValidEmail}
+            />
+            <FormHelperText id="email-helper-text" sx={{ fontColor: 'red' }}>
+              {' '}
+              <StyledErrorText>
+                {isValidEmail ? ' ' : 'Invalid Email'}
+              </StyledErrorText>
+            </FormHelperText>
+          </StyledFields>
+          <Box
+            sx={{
+              marginLeft: 'auto',
+              textAlign: 'center',
+              color: 'purple.100',
+              weight: 600,
+            }}
           >
-            <ArrowForward />
-          </IconButton>
-          <br />
-          Next
-        </Box>
+            <IconButton
+              className="next"
+              onClick={gotoNextStep}
+              disabled={name === '' || email === '' || !isValidEmail}
+              sx={{ boxShadow: 3 }}
+              aria-label="Next button"
+            >
+              <ArrowForward />
+            </IconButton>
+            <br />
+            Next
```

Comment:
I would suggest using `aria-labelledby` instead of `aria-label` here because MDN web docs says "If there is visible text that labels an element, use [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) instead." Would need to put the "Next" in its own element in this case, which is probably better anyway because styling with the `<br />` is not good practice

Was mainly thinking about this for StepTemplate but here too